kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -39,7 +39,7 @@ def _get_synapse_by_name(self, synapse_name):
         """
         all_synapse = self.brain.brain_yaml
         for el in all_synapse:
-            print el
+            print(el)
```

Comment:
And also this print should not be here, it a debug print that should has been pushed into the logger.